### PR TITLE
Rename to Lisk Commander - Closes #440

### LINKED ISSUE #440: Update name to Lisk Commander
**Expected behavior:** We should be using our new name everywhere appropriate.

### PR DESCRIPTION
**Pull request by willclarktech**
### What was the problem?

We hadn't switched to the new name.

### How did I fix it?

I replaced all occurrences of `Lisky` with `Lisk Commander`.

### How to test it?

`npm start`, `npm test`

### Review checklist

* The PR solves #440 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### Additional information

There are some outstanding concerns regarding this PR:

1. CI is not set up to support the name change.
1. GitHub is not set up to support the name change.
1. Should the command be `lisk-commander`? An alternative is simply `lisk`, or something else that's one word.
1. Should the displayed prompt be `lisk-commander`? `lisk` could also make sense there.
1. What file extension are we expecting people to use? I put `.lisk`, but it might be more consistent to use `.lisk-commander` if we end up using that for the command and prompt.

### CODE REVIEW

**Comment by willclarktech** [comment]
Summary after meeting:
1. Add delimiter field to config
1. Delimiter should be `lisk`
1. Name should be `Lisk Commander`
1. Add version field to config
1. Command should be `lisk`

File: README.md [hunk 1]
```diff
@@ -70,23 +70,23 @@ This will also add `lisky` to your PATH, but you won't have to repeat the proces
...
+| |     _ ___| | __ | |     ___  _ __ ___  _ __ ___   __ _ _ __   __| | ___ _ __
+| |    | / __| |/ / | |    / _ \| '_ ` _ \| '_ ` _ \ / _` | '_ \ / _` |/ _ \ '__|
+| |____| \__ \   <  | |___| (_) | | | | | | | | | | | (_| | | | | (_| |  __/ |
+|______|_|___/_|\_\  \_____\___/|_| |_| |_|_| |_| |_|\__,_|_| |_|\__,_|\___|_|
+
+Running v0.3.0.
 Type `help` to get started.
 
-lisky> help
+lisk-commander> help
```
Comment (Tosch110) [inline README.md]: Default delimiter will be `lisk`

Comment (Tosch110) [reply README.md]: This occurs a couple of times in this file

File: README.md [hunk 2]
```diff
@@ -70,23 +70,23 @@ This will also add `lisky` to your PATH, but you won't have to repeat the proces
...
-$ lisky
- _ _     _
-| (_)___| | ___   _
-| | / __| |/ / | | |
-| | \__ \   <| |_| |
-|_|_|___/_|\_\\__, |
-              |___/
-
-Running v0.1.3. Copyright (C) 2017 Lisk Foundation
+$ lisk-commander
```
Comment (Tosch110) [inline README.md]: `lisk-commander` default command will start with `lisk`

File: src/commands/broadcast_signature.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /*
- * LiskHQ/lisky
- * Copyright © 2017 Lisk Foundation
+ * LiskHQ/lisk-commander
+ * Copyright © 2016–2018 Lisk Foundation
```
Comment (Tosch110) [inline src/commands/broadcast_signature.js]: 2016-2018 - even though the Software was first started in 2017 ?

**Review by toschdev — DISMISSED** [review]

File: test/specs/commands/set.js [hunk 4]
```diff
@@ -259,7 +261,7 @@ describe('set command', () => {
 						});
 					});
 					Given('a variable "name"', given.aVariable, () => {
-						Given('a value "my_custom_lisky"', given.aValue, () => {
+						Given('a value "my_custom_lisk_cli"', given.aValue, () => {
```
Comment (shuse2) [inline test/specs/commands/set.js]: `my_custom_lisk_cli ` rather than `my_custom_lisk_commander`?

**Review by shuse2 — DISMISSED** [review]
LGTM

File: test/specs/commands/set.js (hunk 4 above)
Comment (willclarktech) [reply test/specs/commands/set.js]: I thought about this. "Lisk Commander" is our branded Lisk CLI. If people are making their own they might want their own branding.

**Comment by willclarktech** [comment]
Waiting for GitHub/CI before merging.

**Review by toschdev — DISMISSED** [review]

**Comment by willclarktech** [comment]
Suspended until necessary changes have been made on GitHub and Jenkins.

**Review by willclarktech — COMMENTED** [review]
I can only comment as the PR author, but the new FAQ section in the README needs to be updated too.

**Review by willclarktech — COMMENTED** [review]
Approved

**Review by shuse2 — APPROVED** [review]

**Pull request merged**